Jenkinsfile.crossbuild

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env/ groovy
```

Comment:
Rename file from Jenkinsfile.crossbuild to something with platform specific. Also it's good practice to use same platform specific prefix with all related names. For example, "Jenkinsfile.macoslinux.crossbuild" or "Jenkinsfile.macos.crossbuild"

Reply:
Done